rfd/0089-merge-webapps.md

```diff
@@ -0,0 +1,141 @@
+but the final switch over steps will not be taken until v11 has been released.
+
+#### Webapps repository
+
+[repository](https://github.com/gravitational/webapps)
+
+- [ ] Triage issues.
+  - [ ] Close those that are no longer relevant.
+  - [ ] [Transfer remaining issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) to the [Teleport repo](https://github.com/gravitational/teleport)
+  - [ ] Add the [ui label](https://github.com/gravitational/teleport/labels/ui) to
+        those transferred issues.
+- [ ] Triage PRs.
+  - [ ] Close those that are no longer relevant.
+  - [ ] Merge remaining PR's.
+  - [ ] Apply the necessary backports.
+
+#### Webapps.e repository
+
+[repository](https://github.com/gravitational/webapps.e)
+
+- [ ] Triage issues.
+  - [ ] Close those that are no longer relevant.
+  - [ ] [Transfer remaining issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) to the [Teleport.e repo](https://github.com/gravitational/teleport.e)
+  - [ ] Add the [ui label](https://github.com/gravitational/teleport.e/labels/ui)
+- [ ] Triage PRs.
+  - [ ] Close those that are no longer relevant.
+  - [ ] Merge remaining PR's.
+  - [ ] Apply the necessary backports.
+
+#### Teleport repository
+
+[repository](https://github.com/gravitational/teleport)
+
+##### Actions
+
+- Remove `/webassets` submodule
+  - This submodule is no longer required as the web UI will be built on demand.
+- Clone the [Webapps repository](https://github.com/gravitational/webapps) into
+  the Teleport root.
+  - This will need to be done for every respective version branch (v8, v9, v10)
+- update targets that use the `packages/webapps.e` submodule to point points to
+  the correct version in the `teleport.e/webapps` folder.
+- only require teleport build processes to run on teleport paths and the webapp
+  ones to run on the webapp paths
+-
+- Archive the [Webapps repository](https://github.com/gravitational/webapps).
+
+##### CI jobs
+
+- **CodeQL / Analyze (go) (pull_request)**
+- **Teleport-IntegrationTest (ci-account)**
+- **Teleport-UnitTest (ci-account)**
+- **Teleport-DocTest (ci-account)**
+- **Teleport-Lint (ci-account)**
+- **Teleport-OsCompatibility (ci-account)**
+  - Disable for changes exclusively made in the `/webapp` path.
+- **Assign / Auto Request Review**
+  - Create a list of full stack engineers and have it pick from this list for
+    changes made to the `/webapp` path.
+- **Check / Checking reviewers**
+  - Continue to require 2 reviewers for changes in the `webapp` projects.
+- **Label / Label Pull Request (pull_request_target)**
+  - Add label for UI to changes made in the `/webapp` path.
+- **CodeQL / Analyze (javascript) (pull_request)**
+  - Ensure that it's running for changes in `/webapp` path.
+- **Code scanning results / CodeQL**
+  - No changes
+- **webapps-build**
```

Comment:
Would this job go away?

Reply:
Yes it would! Updated.

We should ensure that it can still be built though so we should create a new job to tackle that.